mmv1/third_party/terraform/tests/resource_compute_instance_test.go

```diff
@@ -843,14 +843,18 @@ func TestAccComputeInstance_soleTenantNodeAffinities(t *testing.T) {
 		Providers:    testAccProviders,
 		CheckDestroy: testAccCheckComputeInstanceDestroyProducer(t),
 		Steps: []resource.TestStep{
+			{
+				Config: testAccComputeInstance_withoutNodeAffinities(instanceName, templateName, groupName),
```

Comment:
Can you add this step at the end as well to test that disabling node affinities works?

Reply:
done

Reply:
Turns out adding this step didn't do anything since `scheduling` is computed. I tried out a step that removes the node affinities instead of omits them, but got the error:
`Instances with minNodeCpus must remain sole tenant.`

I added a step to reduce the number of affinities at least